lib/from_openstudio/model.rb

```diff
@@ -239,6 +243,17 @@ def self.constructions_from_model(openstudio_model)
       result
     end
 
+    # Create HB ConstructionSets from OpenStudio Construction Set
+    def self.constructionsets_from_model(openstudio_model)
+      result = []
+
+      openstudio_model.getDefaultConstructionSets.each do |construction_set|
+        result << ConstructionSetAbridged.from_construction_set(construction_set)
```

Comment:
It would be good to just add a check here to make sure that we don't translate the Honeybee "Default Generic Construction Set" since this is hard-coded into honeybee schema and writing it into the HBJSON is going to be redundant. So you would just need a conditional statement like 
```
if construction_set.nameString != "Default Generic Construction Set"
```